components/form/Input.js

```diff
@@ -33,21 +33,21 @@ const Input = ({ className, Icon, labelClassName, name, placeholder, text, type
     <div className="relative z-0 mb-4 w-full group">
       <input
         name={name}
-        placeholder={placeholder ? placeholder : text}
+        placeholder={placeholder ? placeholder : label}
         type={isPasswordField ? (showPassword ? 'text' : 'password') : type}
         id={name}
         {...register(name)}
         className={`${errorClass} block py-2.5 px-0 w-full text-sm  text-amcovad-secondary-700 bg-transparent placeholder-transparent border-2 rounded-md border-amcovad-secondary-300 appearance-none focus:outline-none focus:ring-0  peer ${className}`}
       />
-      <Label htmlFor={name} floatLabel text={text} className={`${errorLabel}${labelClassName}`} />
+      <Label htmlFor={name} floatLabel text={label} className={`${errorLabel}${labelClassName}`} />
```

Comment:
Since we have defined a new component for Label, then it is best for us to move the error implementation into the Label component instead of in the parent component. This is save us from future repetitions and makes the code easier to understand in the future.

```suggestion
      <Label htmlFor={name} floatLabel text={label} className={labelClassName} />
```